hack/update-docs-and-licenses.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright (c) Codesphere Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+repo_root=${1:-$(pwd)}
+
+if [[ ! -f "$repo_root/Makefile" ]]; then
+    echo "ERROR: Makefile not found in $repo_root." >&2
+    exit 2
+fi
```

Comment:
make will fail if it's not here, do we really need an additional check? That's what we have `set -e` for